test/controllers/comments_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class CommentsControllerTest < ActionController::TestCase
+  # test "the truth" do
```

Comment:
The `comment_controller_test` should look something like this:

https://github.com/kenmazaika/yelp-clone/blob/master/test/controllers/comments_controller_test.rb#L16-L29